src/saveload/ser.rs

```diff
@@ -0,0 +1,166 @@
+use std::fmt::Display;
+use std::marker::PhantomData;
+
+use serde::ser::{self, Serialize, SerializeSeq, Serializer};
+
+use {Entities, FetchMut, Join, ReadStorage, WriteStorage};
+
+use saveload::{Components, EntityData, Storages};
+use saveload::marker::{Marker, MarkerAllocator};
+
+/// Serialize components from specified storages via `SerializableComponent::save`
+/// of all marked entities with provided serializer.
```

Comment:
I am not sure what this sentence is trying to say. The "of all marked entities" seems to be dangling. "Serialize all marked entities with provided serializer from specified storages" perhaps?

Reply:
components ... of all marked entities